frontend/components/sections/BitTableSection.tsx

```diff
@@ -111,55 +100,140 @@ function Bit({ type, driverSizeColor, sizes }: GroupedBit) {
                height={20}
             />
-            <Flex
-               px="1"
-               h="4"
-               bg={`${driverSizeColor}.100`}
-               borderRadius="full"
-               borderColor={`${driverSizeColor}.200`}
-               borderWidth={1}
-               alignItems="center"
-               justify="center"
-               minW="10"
-            >
-               <Text
-                  as="span"
-                  color={`${driverSizeColor}.700`}
-                  fontSize="xs"
-                  fontFamily="mono"
-               >
-                  {type.driverSize}
-               </Text>
-            </Flex>
-         </Flex>
-         <Flex flexGrow={1} alignItems="center" gap="2.5" m="2.5">
-            <Text flexGrow={1}>{type.name}</Text>
-            <Flex flexWrap="wrap" gap={1} justifyContent="flex-end">
-               {sizes.map((size) => (
+            <Flex flexDirection="column" gap="1">
+               {Object.keys(sizes).map((driverSize) => (
                   <Flex
-                     key={size}
-                     px="1.5"
+                     key={driverSize}
+                     px="1"
                      h="4"
-                     bg="brand.100"
+                     bg={`${driverSizeColors[driverSize]}.100`}
                      borderRadius="full"
-                     borderColor="brand.200"
+                     borderColor={`${driverSizeColors[driverSize]}.200`}
                      borderWidth={1}
                      alignItems="center"
                      justify="center"
-                     minW="9"
+                     minW="10"
                   >
                      <Text
                         as="span"
-                        color="brand.700"
+                        color={`${driverSizeColors[driverSize]}.700`}
                         fontSize="xs"
                         fontFamily="mono"
                      >
-                        {size}
+                        {driverSize}
                      </Text>
                   </Flex>
                ))}
             </Flex>
          </Flex>
+         <Flex flexGrow={1} alignItems="center" gap="3" p="3">
+            <Flex gap="1">
+               {totalBitCount > 0 && (
+                  <>
+                     <Text flexGrow={1}>{totalBitCount}</Text>
+                     <Text color="gray.400">×</Text>
+                  </>
+               )}
+               <Text>{type.name}</Text>
+            </Flex>
+
+            <Flex flexGrow="1" flexDirection="column" gap={1}>
+               {Object.keys(sizes).map((driverSize) => {
+                  const sizesForDriver = sizes[driverSize];
+                  return (
+                     <Flex
+                        key={driverSize}
+                        flexWrap="wrap"
+                        gap={1}
+                        justifyContent="flex-end"
+                     >
+                        {sizesForDriver.map((size) => (
```

Comment:
If `size` was null for all bits with this `driverSize`, we could have `sizesForDriver === []`. Do we still want to be rendering the parent `Flex` in that case? If not, we might be able to simplify the grouping logic, since we don't need to deal with quite as many `null` checks.

Reply:
We can omit the div